torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1223,3 +1225,12 @@ def call_function(
             ),
             example_value=None,
         )
+
+
+class TraceWrappedHigherOrderOperatorVariable(TorchHigherOrderOperatorVariable):
+    def call_function(
+        self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
+    ) -> "VariableTracker":
+        assert "fn" in kwargs
+        fn = kwargs["fn"]
+        return fn.call_function(tx, args, {})
```

Comment:
maybe add a comment like `# just inline into the function, Dynamo doesn't do anything special with the higher order operator`

Reply:
sgtm